src/components/dashboard/TaskList.js

```diff
@@ -37,9 +37,15 @@ const TaskList = () => {
       <Search />
       <div
         sx={{
-          display: `grid`,
+          // display: `grid`,
```

Comment:
The `display: grid` still needs to be here so the task cards don't get too big for the mobile view